src/store/data_layer/data_store.js

```diff
@@ -33,12 +33,168 @@ const state = () => {
       //   }
       // }
     },
-    localStorageChecked: false
+    indexedDBInstance: undefined,
+    remoteSystemNow: undefined
   }
 }
 
 const actions = {
 
+  // Queries backend-api to get "system.now()" timestamp
+  // this is to ensure DeadRequests doesn't improperly
+  // mark intervals as "unavailable."
+  async loadSystemNow (store) {
+    let apiTime = await API.systemtime()
+    // see if we can set the apiTime from querying AWS
+    if (apiTime) {
+      this.commit('dataStore/setSystemNow', { now: Number(apiTime) })
+    } else {
+      // otherwise, fall-back to using locale time
+      this.commit('dataStore/setSystemNow', { now: Date.now() })
```

Comment:
I think we should `console.log()` something or throw some sort of exception, in addition to what you already have. That way, there is a higher likelihood we'll discover issues with the date api route.